process.go

```diff
@@ -219,9 +260,17 @@ func (c processPortsChecker) Coerce(v interface{}, path []string) (interface{},
 		return nil, fmt.Errorf("%s: invalid value %q", strings.Join(path[1:], ""), item)
 	}
 
-	portA, err := strconv.Atoi(parts[0])
-	if err != nil {
-		return nil, fmt.Errorf("%s: expected int got %q", strings.Join(path[1:], ""), parts[0])
+	portA := 0
+	external := parts[0]
+	if !strings.HasPrefix(external, "<") || !strings.HasSuffix(external, ">") {
```

Comment:
It's not immediately evident what this chunk of code is trying to accomplish. Please add a few pointer comments.